verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -42,8 +43,12 @@ class KytheFactsPrinter {
   const IndexingFactNode& root_;
 };
 
+std::ostream& operator<<(std::ostream&, const KytheFactsPrinter&);
+
 // Responsible for traversing IndexingFactsTree and processing its different
 // nodes to produce kythe indexing facts.
+// Iteratively extracts facts and keeps running till no new facts are found in
```

Comment:
Iteratively: can you bound the number of iterations?  Is this truly a fixed-point calculation?